bpfctl/src/main.rs

```diff
@@ -69,6 +69,12 @@ enum LoadCommands {
         /// Required: Priority to run program in chain. Lower value runs first.
         #[clap(long)]
         priority: i32,
+        /// Optional: Proceed to call other programs in chain on this exit code.
+        /// Multiple values supported by repeating the parameter.
+        /// Possible values: [aborted, drop, pass, tx, redirect, dispatcher_return]
```

Comment:
Replace the last two lines of description with "Not currently support on TC programs."

Reply:
Ok.